include/native/parallel_execution_native.h

```diff
@@ -435,11 +435,11 @@ class parallel_execution_native {
       mpmc_queue<output_type> & output_queue) const;
 
   template <typename T, typename ... Others>
-  void do_pipeline(mpmc_queue<T> & in_q, mpmc_queue<T> & same_queue, Others &&... ops) const
+  void do_pipeline(mpmc_queue<T> &, mpmc_queue<T> &, Others &&...) const
```

Comment:
OK. Unused, but this is going to complicate documentation.

Suggestion:
1.- Leave parameter names in declaration.
2.- Add a definition without parameter names.

Reply:
Separated definition (with names for documentation) and definition without them, to avoid warnings

Reply:
OK